app/models/suggestion.py

```diff
@@ -0,0 +1,120 @@
+from .. import db
+from datetime import datetime
+import pytz
+
+
+class Suggestion(db.Model):
+    """
+    Association between a resource and potential suggestions for it
+    """
+    __tablename__ = 'suggestions'
+    id = db.Column(db.Integer, primary_key=True)
+    # suggestion_type is 0 for insertion, 1 for edit, and 2 for deletion
+    suggestion_type = db.Column(db.Integer)
+    resource_id = db.Column(db.Integer)
+    suggestion_text = db.Column(db.String(150))
+    # 0 stands for read, 1 stands for unread
+    read = db.Column(db.Integer, default=0)
+    timestamp = db.Column(db.DateTime)
+    contact_name = db.Column(db.String(64))
+    contact_email = db.Column(db.String(64), unique=True, index=True)
```

Comment:
Why `unique=True` and `index=True`? What if the same person makes multiple suggestions and uses the same email address?